include/SDL3/SDL_gpu.h

```diff
@@ -1178,84 +1178,84 @@ typedef struct SDL_GPUTransferBufferCreateInfo
-    SDL_bool depthBiasEnable;
+    SDL_bool enable_depth_bias;
     Uint8 padding1;
     Uint8 padding2;
     Uint8 padding3;
-    float depthBiasConstantFactor;
-    float depthBiasClamp;
-    float depthBiasSlopeFactor;
+    float depth_bias_constant_factor;
+    float depth_bias_clamp;
+    float depth_bias_slope_factor;
 } SDL_GPURasterizerState;
 
 typedef struct SDL_GPUMultisampleState
 {
-    SDL_GPUSampleCount sampleCount;
-    Uint32 sampleMask;
+    SDL_GPUSampleCount sample_count;
+    Uint32 sample_mask;
 } SDL_GPUMultisampleState;
 
 typedef struct SDL_GPUDepthStencilState
 {
-    SDL_bool depthTestEnable;
-    SDL_bool depthWriteEnable;
-    SDL_bool stencilTestEnable;
+    SDL_bool enable_depth_test;
+    SDL_bool enable_depth_write;
+    SDL_bool enable_stencil_test;
     Uint8 padding1;
-    SDL_GPUCompareOp compareOp;
-    SDL_GPUStencilOpState backStencilState;
-    SDL_GPUStencilOpState frontStencilState;
-    Uint8 compareMask;
-    Uint8 writeMask;
+    SDL_GPUCompareOp compare_op;
+    SDL_GPUStencilOpState back_stencil_state;
+    SDL_GPUStencilOpState front_stencil_state;
+    Uint8 compare_mask;
+    Uint8 write_mask;
     Uint8 padding2;
     Uint8 padding3;
 } SDL_GPUDepthStencilState;
 
 typedef struct SDL_GPUColorAttachmentDescription
 {
     SDL_GPUTextureFormat format;
-    SDL_GPUColorAttachmentBlendState blendState;
+    SDL_GPUColorAttachmentBlendState blend_state;
 } SDL_GPUColorAttachmentDescription;
 
 typedef struct SDL_GPUGraphicsPipelineAttachmentInfo
 {
-    const SDL_GPUColorAttachmentDescription *colorAttachmentDescriptions;
-    Uint32 colorAttachmentCount;
-    SDL_bool hasDepthStencilAttachment;
+    const SDL_GPUColorAttachmentDescription *color_attachment_descriptions;
+    Uint32 num_color_attachments;
+    SDL_bool has_depth_stencil_attachment;
     Uint8 padding1;
     Uint8 padding2;
     Uint8 padding3;
-    SDL_GPUTextureFormat depthStencilFormat;
+    SDL_GPUTextureFormat depth_stencil_format;
 } SDL_GPUGraphicsPipelineAttachmentInfo;
 
 typedef struct SDL_GPUGraphicsPipelineCreateInfo
 {
-    SDL_GPUShader *vertexShader;
-    SDL_GPUShader *fragmentShader;
-    SDL_GPUVertexInputState vertexInputState;
-    SDL_GPUPrimitiveType primitiveType;
-    SDL_GPURasterizerState rasterizerState;
-    SDL_GPUMultisampleState multisampleState;
-    SDL_GPUDepthStencilState depthStencilState;
-    SDL_GPUGraphicsPipelineAttachmentInfo attachmentInfo;
+    SDL_GPUShader *vertex_shader;
+    SDL_GPUShader *fragment_shader;
+    SDL_GPUVertexInputState vertex_input_state;
+    SDL_GPUPrimitiveType primitive_type;
+    SDL_GPURasterizerState rasterizer_state;
+    SDL_GPUMultisampleState multisample_state;
+    SDL_GPUDepthStencilState depth_stencil_state;
+    SDL_GPUGraphicsPipelineAttachmentInfo attachment_info;
 
     SDL_PropertiesID props;
 } SDL_GPUGraphicsPipelineCreateInfo;
 
 typedef struct SDL_GPUComputePipelineCreateInfo
 {
-    size_t codeSize;
+    size_t code_size;
```

Comment:
Uint32 size?
Maybe this field should come after code?

Reply:
"size" feels ambiguous to me. "num_code_bytes" might be better.